repos/system_upgrade/common/actors/dnfconfuserspacecopy/actor.py

```diff
@@ -0,0 +1,24 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import dnfconfuserspacecopy
+from leapp.models import TargetUserSpacePreupgradeTasks
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class DNFConfUserspaceCopy(Actor):
+    """
+    Copy dnf.conf to target userspace
+
+    Copies /etc/leapp/files/dnf.conf to target userspace. If it isn't available
+    /etc/dnf/dnf.conf is copied instead. This allows specifying a different
+    config for the target userspace, which might be required if the source
+    system configuration file isn't compatible with the target one. One such
+    example is incompatible proxy configuration between RHEL7 and RHEL8 DNF
+    versions.
+    """
+    name = "dnf_conf_userspace_copy"
```

Comment:
The name is not descriptive, similarly for the other introduced actor.  
```suggestion
    name = "copy_dnf_conf_into_target_userspace"
```

Reply:
I agree.